### PR TITLE
Tines Webinar Prep: Okta and CloudTrail Rules/Scenario

### PR DESCRIPTION
**Pull request by jacknagz**
### Background

Two sets of scenarios and rules for the upcoming Panther+Tines webinar!

### Changes

* Basic Okta rules
* Update CloudTrail dedup and alert contexts

### Testing

* Unit tests + full end-to-end testing with the `send_data` script (we should fold this functionality into the PAT at some point)

### CODE REVIEW

**Comment by jacknagz** [comment]
Looks like CI is broken:

```
ERROR: Cannot install -r requirements.txt (line 16) and aws-lambda-builders==0.6.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested aws-lambda-builders==0.6.0
    aws-sam-cli 0.31.1 depends on aws-lambda-builders==0.5.0
```

**Review by nhakmiller — APPROVED** [review]

**Comment by wey-chiang** [comment]
> Looks like CI is broken:
> 
> ```
> ERROR: Cannot install -r requirements.txt (line 16) and aws-lambda-builders==0.6.0 because these package versions have conflicting dependencies.
> 
> The conflict is caused by:
>     The user requested aws-lambda-builders==0.6.0
>     aws-sam-cli 0.31.1 depends on aws-lambda-builders==0.5.0
> ```

Looks like `aws-sam-cli 0.31.1` was released back in Nov 2019, easiest way to resolve the conflicting deps is to upgrade the `aws-sam-cli` package

File: aws_cloudtrail_rules/aws_console_root_login.py [hunk 1]
```diff
@@ -11,3 +11,19 @@ def title(event):
...
+
+
+def alert_context(event):
+    return {
+        'sourceIPAddress': event['sourceIPAddress'],
+        'userIdentityAccountId': event['userIdentity']['accountId'],
+        'userIdentityArn': event['userIdentity']['arn'],
+        'eventTime': event['eventTime'],
+        'mfaUsed': event.get('additionalEventData', {}).get('MFAUsed')
+    }
```
Comment (lindsey-w) [inline aws_cloudtrail_rules/aws_console_root_login.py:29]: This is great!

**Review by lindsey-w — APPROVED** [review]
I think a `make deps-update` will update the requirements file and fix the ci issue. It removes the clashing ones altogether for me.

**Comment by jacknagz** [comment]
@lindsey-w @wey-chiang can one of you pls fix the dependency issue and I'll rebase?

File: aws_cloudtrail_rules/aws_console_root_login.py (hunk 1 above)
Comment (jacknagz) [reply aws_cloudtrail_rules/aws_console_root_login.py:29]: Yeah! We can even make it a helper if we like it

**Pull request merged**